modules/AWS/eks/main.tf

```diff
@@ -22,6 +32,8 @@ module "eks" {
   kubeconfig_aws_authenticator_command      = "aws"
   kubeconfig_aws_authenticator_command_args = ["eks", "get-token", "--cluster-name", var.cluster_name]
 
+  workers_additional_policies = var.secret_id != "" ? [aws_iam_policy.laas[0].arn,aws_iam_policy.fleet_enrollment_secret[0].arn] : []
```

Comment:
This conditional can be computed in `locals.tf`.

Reply:
Good idea, moved to locals.